composer.json

```diff
@@ -26,21 +25,63 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "composer/installers": "^1.2",
-        "cweagans/composer-patches": "^1.6.5",
-        "dof-dss/nicsdru_origins_profile": "dev-development",
-        "drupal-composer/drupal-scaffold": "^2.5",
-        "drupal/console": "^1.0.2",
-        "drupal/core": "^8.7.0",
-        "drush/drush": "^9.0.0",
-        "vlucas/phpdotenv": "^2.4",
-        "webflo/drupal-finder": "^1.0.0",
-        "webmozart/path-util": "^2.3",
-        "zaporylie/composer-drupal-optimizations": "^1.0"
+        "composer/installers": "^1.9",
+        "cweagans/composer-patches": "^1.7",
+        "dof-dss/nicsdru_origins_modules": "^0.9",
+        "dof-dss/nicsdru_origins_profile": "^0.1",
+        "drupal/admin_toolbar": "^2.0",
+        "drupal/adminimal_theme": "^1.6",
+        "drupal/config_filter": "^1.4.0",
```

Comment:
I'd update config filter to the 2.2 release.

Reply:
Yep, good point. I'd shamelessly nabbed from Unity but with a little composer adjustments these now use more recent versions in the template repo.